packages/trpc/server/routers/viewer/eventTypes.ts

```diff
@@ -114,6 +115,11 @@ const EventTypeUpdateInput = _EventTypeModel
   .partial()
   .extend({
     metadata: EventTypeMetaDataSchema.optional(),
+    bookingFields: eventTypeBookingFields.element
+      .omit({
+        fieldTypeConfig: true,
```

Comment:
We don't want to save fieldTypeConfig. It is there in bookingFields as a quick way to access the corresponding field type's config